### DISCUSSION TITLE
What does the option "Cadence sensor as a bike" do?

### DISCUSSION THREAD

**Discussion by lychee128** [cagnulein/qdomyos-zwift#308] [Q&A] [answered]
Does this mean that the app reads the cadence info from an external sensor (eg, Wahoo) rather than the bike’s internal computer?  Or is this for some other purpose?  Thanks.

**Comment by cagnulein** [comment, marked as answer]
This option is useful for dumb bikes (bikes without Bluetooth). With this setting to ON you're able to use your cadence sensor as a bike instead of a simple cadence sensor. So you will see all the metrics

**Reply by lychee128** [reply]
Thanks! If my Wahoo external cadence sensor is more accurate than my bike’s built in sensor, can I turn on this setting for your app to use the external sensor rather than the bike’s? And if so, will all the other readings still come from the bike? I only want the cadence rate to be used.

**Reply by cagnulein** [reply]
Not yet but it's absolutely possibile. Open a bug request about this. I will be happy to add it (honestly another user asked me the same thing yesterday)

**Reply by lychee128** [reply]
Thanks and will do.
